File: cpp/model/Item.cpp
#include "Item.h"

Item::Item(std::string & name, int baseDamage, float damageModifier) : name(name), baseDamage(baseDamage), damageModifier(damageModifier) {}

const float Item::getDamageModifier() {
    return damageModifier;
}

const int Item::getBaseDamage() {
    return baseDamage;
}

File: python/Item.py
class Item:
    base_damage: int
    damage_modifier: float


class BaseItem(Item):
    name: str

    def __init__(
        self,
        name: str,
        base_damage: int,
        damage_modifier: float
    ) -> None:
        self.name = name
        self.base_damage = base_damage
        self.damage_modifier = damage_modifier
